CMakeLists.txt

```diff
@@ -44,11 +44,13 @@ set(TEST_LIBS ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} ${Boost_LIBRARIES})
 
 # Workaround
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.5.0)
-  else()
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 3.5.0)
```

Comment:
I'm afraid the QA tests only run when CRUX is included as part of the MaidSafe project, so this part of the CMakeLists.txt isn't tested by QA.

We'd need to build CRUX as a standalone project to have the tests hit this bit, so I think @breese's concern is valid.  Also, checking for versions being *equal to* a specific version rather than less/greater than leaves the possibility for errors. (e.g. how would we handle v3.4.1?)